### ISSUE TITLE
./rustscan 1.1.1.1 bash: ./rustscan: No such file or directory

### ISSUE THREAD

**Issue by denisgolius** [RustScan/RustScan#44]
```
ubuntu18-04:/tmp# ls -1 | grep rust
rustscan
ubuntu18-04:/tmp# ll -1 | grep rust
-rwxr-xr-x  1 dengolius 1001 3957296 Jul 23 12:06 rustscan*
ubuntu18-04:/tmp# ./rustscan 1.1.1.1
bash: ./rustscan: No such file or directory
ubuntu18-04:/tmp# rm -rf /usr/local/bin/rustscan_1.1.0_amd64.deb 
ubuntu18-04:/tmp# rm -rf /usr/local/bin/rustscan
ubuntu18-04:/tmp# chmod +x rustscan 
ubuntu18-04:/tmp# ./rustscan 1.1.1.1
bash: ./rustscan: No such file or directory
```

my system
```
ubuntu18-04:/tmp# cat /etc/*release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.4 LTS"
NAME="Ubuntu"
VERSION="18.04.4 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.4 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
ubuntu18-04:/tmp# uname -a
Linux px4-zabbix 4.15.0-108-generic #109-Ubuntu SMP Fri Jun 19 11:33:10 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
ubuntu18-04:/tmp# 
```

**Comment by SakiiR** [comment]
Can you provide the output of `file ./rustscan` please

**Comment by denisgolius** [comment]
Issue fixed in 1.2.1 version.
